src/store/CitySlice.tsx

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const citySlice = createSlice({
+    name: 'City',
+    initialState: {
+        cities: [],
+    },
+    reducers: {
+        addCity(state, action) {
+            state.cities.push({
+                location: action.payload.text,
+                timeZone: action.payload.text,
```

Comment:
If `text` field isn't a placeholder for test, you should remove it and replace with normal city model